make/Makefile-local-install

```diff
@@ -25,6 +25,7 @@ mdai-system-reqs:
 mdai-add-helm-charts:
 	@echo "🟢 Start mdai-add-helm-charts..."
 	@helm repo list | grep prometheus-community || helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+	@helm repo list | grep open-telemetry || helm repo add open-telemetry https://decisiveai.github.io/mdai-helm-charts
```

Comment:
open-telemetry could collide with the common naming for the already installed OTEL Helm chart, better use mdai-*

Reply:
why not utilize the existing `mydecisive` repo?